composer.json

```
@@ -25,11 +25,11 @@
         "sensiolabs/security-checker": " ^6.0",
         "sylius/sylius": "dev-master",
         "symfony/flex": "^1.7",
-        "sylius/invoicing-plugin": "^0.14",
-        "sylius/refund-plugin": "^1.0.0-RC.10",
+        "sylius/invoicing-plugin": "^0.17",
```

To not to block on only one version:
```suggestion
        "sylius/invoicing-plugin": "~0.17",
```